src/content/1.7/modules/sell/techvalidation-checklist.md

```diff
@@ -34,6 +34,14 @@ Additionally, distribution licenses like CC-0 or CC-by-sa are appropriate for ar
 
 Module may create all the table they need in the database. However altering core table is forbidden.
 
+#### Other modules are not altered
+
+Modifying core or other modules files is not allowed.
+
+#### File storage in proper directory
+
+Module may add / modify some files on the shop. To avoid issues with file permissions, we recommend storing files in the `var/%env%` dirrectory.
```

Comment:
```suggestion
Module may add / modify some files on the shop. To avoid issues with file permissions, we recommend storing files in the `var/%env%` directory.
```